### ISSUE TITLE
Topic facet buttons not accessible to screenreader users

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#145]
https://nara-bia-photos.herokuapp.com/topics/buildings?tribalNations[0]=Confederated%20Salish%20&%20Kootenai%20Tribes%20of%20the%20Flathead%20Reservation,%20Montana&tribalNations[1]=Crow%20Creek%20Sioux%20Tribe&tribalNations[2]=Duckwater%20Shoshone%20Tribe%20of%20the%20Duckwater%20Reservation,%20Nevada

Search facets not identified as such and therefore are not accessible to screenreader users who likely will not understand their function, adding some descriptive aria labels to the buttons would solve the problem.

![image](https://user-images.githubusercontent.com/6942589/96005637-a1167f00-0e0a-11eb-8d74-2394623a68dd.png)
